src/NServiceBus.AcceptanceTests/ScenarioDescriptors/TypeScanner.cs

```diff
@@ -4,7 +4,7 @@ namespace NServiceBus.AcceptanceTests.ScenarioDescriptors
     using System.Collections.Generic;
     using System.Linq;
     using System.Reflection;
-    using NServiceBus.Hosting.Helpers;
+    using Hosting.Helpers;
```

Comment:
I think because we removed the hosting folder which removes the need to fully qualify the namespace in this case.